core/domain/question_services_test.py

```diff
@@ -129,9 +129,9 @@ def test_create_and_get_question_skill_link(self):
             if (
                     question_ids[index] == self.question_id or
                     question_ids[index] == question_id_2):
-                self.assertEqual('Skill Description 1', description)
+                self.assertEqual(['Skill Description 1'], description)
```

Comment:
Test a case where we get multiple skill descriptions

Reply:
Done.

Reply:
Sorry where was this done? I Noticed it in topic_editor_test.py but cant seem to see a case in question_services_test.py..Can you add one in this unit test suite as well?

Reply:
Is this resolved or not? I see a last comment by @vinitamurthi but no follow-up.

Reply:
Done.